src/plugin.js

```diff
@@ -199,7 +210,7 @@ function doZoom(chart, percentZoomX, percentZoomY, focalPoint, whichAxes) {
 
 		// Which axe should be modified when figers were used.
 		var _whichAxes;
-		if (zoomMode === 'xy' && whichAxes !== undefined) {
+		if (directionEnabled(zoomMode, 'x') && directionEnabled(zoomMode, 'y') && whichAxes !== undefined) {
```

Comment:
We need to support `panMode` on line 315 as well, right? 

https://github.com/chartjs/chartjs-plugin-zoom/blob/b5666c46e109119d1875fe3fc59f3b91bf6bf6e3/src/plugin.js#L315